relay-general/src/store/normalize/contexts.rs

```diff
@@ -8,14 +8,17 @@ lazy_static::lazy_static! {
     static ref OS_WINDOWS_REGEX1: Regex = Regex::new(r#"^(Microsoft )?Windows (NT )?(?P<version>\d+\.\d+\.\d+).*$"#).unwrap();
     static ref OS_WINDOWS_REGEX2: Regex = Regex::new(r#"^Windows\s+\d+\s+\((?P<version>\d+\.\d+\.\d+).*$"#).unwrap();
 
-    static ref OS_ANDROID_REGEX: Regex = Regex::new(r#"^Android OS (?P<version>(\d+)) / API-(?P<api>(\d+))"#).unwrap();
+    static ref OS_ANDROID_REGEX: Regex = Regex::new(r#"^Android (OS )?(?P<version>\d+(\.\d+){0,2}) / API-(?P<api>(\d+))"#).unwrap();
```

Comment:
isn't a breaking change? I believe we can see both use cases here, `Android OS...` and `Android (OS ...)` with or without `()`

Reply:
This regex accepts strictly more values than before is all I can say. What is a breaking change in this context?

Reply:
if its more inclusive, all good.
Regex isn't my best skill.

Reply:
It now accepts both `Android ` and `Android OS `, then also some extra segments in the version number